cpp/unreal_projects/SpearSim/Python/validate_spear_scene.py

```diff
@@ -0,0 +1,579 @@
+import argparse
+import numpy as np
+import pathlib
+import posixpath
+import unreal
+import spear
+
+
+def validate_level(level):
+    #
+    # TODO:
+    # We name each scene using a lower_case_with_underscore naming convention and a four
+    # digit suffix (e.g., "apartment_0000", "apartment_0001", etc).
+    #
+
+    pass
+
+
+def validate_content_browser(level):
+    #
+    # TODO:
+    # There should be exactly one umap file with the same name as the scene in the scene's
+    # Maps directory. Our code imposes this requirement to support loading scenes by name.
+    #
+
+    #
+    # TODO:
+    # The "Content Browser" should not contain unreferenced assets, except for in the
+    # "Debug" directory.
+    #
+
+    pass
+
+
+def validate_outliner(level):
+    editor_actor_subsystem = unreal.get_editor_subsystem(unreal.EditorActorSubsystem)
+
+    actors = editor_actor_subsystem.get_selected_level_actors()
```

Comment:
I'm not in favor of this assumption. I might want to validate all actors, even if I have some actors selected. I might have them selected because I'm doing something else. If a user wants to validate selected actors, they should do so with a command-line argument passed into this script.